### PR TITLE
feat(ui): show loading badge when the runtime is in loading status

### LINKED ISSUE #751: Fix loading badge in runtime info panel
When the runtime is loading, the runtime info panel shows the `Running` badge. We need to update the badge when the runtime is loading to show the `Loading` status.

### PR DESCRIPTION
**Pull request by rrbarrero**
We show feedback to the user when the runtime is still being deployed and is not ready yet by adding a loading tag in the runtimeInfo component

closes #751

### CODE REVIEW

**Comment by alejandrogr** [comment]
Very nice addition @rrbarrero ! 

Please we try to follow angular semmantics in commits/PRs title. Change the title to something like: `feat(ui): show loading badge when the runtime is in loading status`

**Comment by rrbarrero** [comment]
> Very nice addition @rrbarrero !
> 
> Please we try to follow angular semmantics in commits/PRs title. Change the title to something like: `feat(ui): show loading badge when the runtime is in loading status`

Done. Thank you for the feedback @alejandrogr ...

File: app/ui/src/Pages/Project/panels/RuntimeInfo/RuntimeInfo.module.scss [hunk 1]
```diff
@@ -30,6 +30,22 @@
       font-weight: bold;
       max-width: 13 *$grid-unit;
     }
+
+    .loadingTag {
```
[the remark this answers is not in the document]
Comment (rrbarrero) [reply app/ui/src/Pages/Project/panels/RuntimeInfo/RuntimeInfo.module.scss:34]: Done. Thank you @JYisus

File: app/ui/src/Pages/Project/panels/RuntimeInfo/RuntimeInfo.module.scss [hunk 2]
```diff
@@ -30,6 +30,22 @@
...
+      @include font-small;
+      display: inline;
+      position: absolute;
+      margin-left: 2*$grid-unit;
+      justify-self: center;
+      margin-top: 0.75 * $grid-unit;
+      text-align: center;
+      padding: 0.4 * $grid-unit 3.5 * $grid-unit;
+      background-color: palette(feedback, 700);
```
[the remark this answers is not in the document]
Comment (rrbarrero) [reply app/ui/src/Pages/Project/panels/RuntimeInfo/RuntimeInfo.module.scss]: Done, thanks mate.

File: app/ui/src/Pages/Project/panels/RuntimeInfo/RuntimeInfo.module.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (rrbarrero) [reply app/ui/src/Pages/Project/panels/RuntimeInfo/RuntimeInfo.module.scss:34]: Done. Thank you @alejandrogr

**Review by alejandrogr — APPROVED** [review]

File: app/ui/cypress/integration/runtimes.spec.ts [hunk 3]
```diff
@@ -158,6 +166,22 @@ describe('Runtimes Behaviour', () => {
       // THEN should redirect to overview page
       cy.url().should('contain', '/overview');
     });
+
+    it.only('should show the badged loading or running', () => {
```
Comment (JYisus) [inline app/ui/cypress/integration/runtimes.spec.ts:170]: @rrbarrero Delete the .only once you end test to allow the execution of the rest of the tests.

Comment (rrbarrero) [reply app/ui/cypress/integration/runtimes.spec.ts]: My mistake, sorry.

**Pull request merged**